src/destructors.md

```diff
@@ -346,6 +413,8 @@ let x = (&temp()).use_temp();  // ERROR
 
 ## Not running destructors
 
+r[destructors.forget]
+
```

Comment:
Let's be consistent on whether or not there's a line break between the `r[..]` tag and the content that follows when it's ahead of a paragraph like this.

Reply:
The tag here is supposed to be attached to the heading. When the content in the section is small enough to not require more than one of its own tags, I've just omitted the tags on the paragraphs themselves.

Reply:
Makes sense.  Seems reasonable.